Lab Member YAMLs/Templates/Full Name with Spaces.yaml

```diff
@@ -0,0 +1,46 @@
+name:
+birthday: MM.DD
+position: (Job Title) (Graduate Student) (Postoctoral Research Scholar/Fellow)
+affiliations: >
+  Weill Cornell Physiology, Biophysics, and Systems Biology PhD Program
+  Weill Cornell Biochemistry and Structural Biology PhD Program
+  Tri-Institutional PhD Program in Computational Biology and Medicine
+  Tri-Institutional PhD Program in Chemical Biology
+  Louis V. Gerstner, Jr. Graduate School of Biomedical Sciences
+  Berlin Institute of Health (BIH) Einstein Foundation - Stiftung Charité Group
+  Joint Mentorship with Name (URL to lab)
+fellowships: >
+ (Any awards you have won) (i.e. MolSSI Investment Fellowship)
+funding: >
+ (list of funding sources)
+status: current
```

Comment:
This syntax is invalid, and the date format needs to be easily parseable in Python.

How about
```yaml
# Key dates. Comment out dates that don't apply to you.
dates:
  start: YYYY-MM-DD
  ace: YYYY-MM-DD
  committee-meetings:
    - YYYY-MM-DD
    - YYYY-MM-DD
  end: YYYY-MM-CC
```